scripts/mprocs.sh

```diff
@@ -12,7 +12,8 @@ export FM_VERBOSE_OUTPUT=0
 
 source scripts/build.sh
 
-devimint dev-fed 2>/dev/null &
+mkdir -p $FM_LOGS_DIR
+devimint dev-fed 2>$FM_LOGS_DIR/devimint-outer.log &
```

Comment:
tmuxinator, too?

Reply:
I wonder why this shows error output than `devimint.log` does? It's kind of confusing that we have 2 different log files. Ideally we could just get the errors to show up in `devimint.log` ...

Reply:
Problem with having devimint write to `devimint.log` internally is that it doesn't catch panics -- which are are pretty imporant!